opentegra.conf

```diff
@@ -3,3 +3,10 @@ Section "OutputClass"
     MatchDriver "tegra"
     Driver "opentegra"
 EndSection
+
```

Comment:
I think you have to comment the whole Screen section.
The purpose of the opentegra.conf snipet is to be installed in /usr/share/X11/xorg.conf.d directory along others vendor ddx files. 
You cannot assume the screen will be a tegra screen if using nxp or broadcom soc etc.
Eventually such options can be made if the files is copied into /etc/X11/xorg.conf.d and changed in a per-system basis by the system administrator or user.

Reply:
But better would be to move the option into the OutputClass section ? Is there any issue doing that ?

Reply:
OutputClass isn't the right place for these options, you'll get invalid config error from Xorg if you'll try to do it.

Screen section won't be used by anything other than opentegra because it has Identifier that links it to correct driver. But you are right, should be better to comment out it completely .

Also I'm thinking again that these options should be in the Device section, even if they are parsed from Screen section too.

Reply:
At least man intel seems to have related option that might be useful by the driver to control which AccelMethod or if Accel is fully disabled.

Good for commenting the whole screen section in case where option are only relevant on a per-screen basis.

Reply:
Indeed, I've commented out it now, thank you for the suggestion. Also, I changed config option name to something that sounds more generic, like "NoAccel" is what Nouveau uses.